gnomad_mitochondria/manuscript_analyses/generate_Fig1A_4_3BCD_S4ABC_S5.R

```diff
@@ -0,0 +1,682 @@
+#!/usr/bin/env Rscript
+
+library(cowplot)
+library(dplyr)
+library(ggplot2)
+library(grid)
+library(optparse)
+library(tidyr)
+library(stringr)
+
+# Set up directories
+option_list = list(
+  make_option(c("-s", "--sample_annotations"), type = "character", default = NULL,
+              help = "The sample annotations file output by add_annotations.py for release samples", metavar = "character"),
+  make_option(c("-u", "--unfiltered_sample_annotations"), type = "character", default = NULL,
+              help = "Sample annotations for all, including filtered out samples, only needs to include mt_mean_coverage column", metavar = "character"),
```

Comment:
Following the edits, this unfiltered file is not used in this script and could be removed here.